src/types/index.d.ts

```diff
@@ -2,6 +2,13 @@
 export {};
 
 declare global {
+  interface Window {
+		darkMode: { 
+                  toggle: () => void 
```

Comment:
If the function is expected to return a result, then I would type this with the value of the expected result.  

```suggestion
                  toggle: () => boolean 
```

Reply:
I fixed the typings here: https://github.com/simonv3/beam/pull/65/commits/393e021d98f2e5d971745d87b7f4e687d2df8301

However, `darkMode` is still not present on the `window` object when called, here's the errors we get on button press:
<img width="1680" alt="Screen Shot 2022-04-24 at 9 43 36 AM" src="https://user-images.githubusercontent.com/60944077/164982054-b7f8ac75-f800-49b5-a891-738802f878e5.png">